pkg/apiserver/apiserver.go

```diff
@@ -74,9 +74,9 @@ func init() {
 
 // monitor is a helper function for each HTTP request handler to use for
 // instrumenting basic request counter and latency metrics.
-func monitor(handler string, verb, resource *string, httpCode *int, reqStart time.Time) {
```

Comment:
If we're removing handler from the fields, it should be removed from this function and its callers as well